crates/rome_cli/src/lib.rs

```diff
@@ -89,6 +89,7 @@ impl<'app> CliSession<'app> {
 
             Some("start") => commands::daemon::start(self),
             Some("stop") => commands::daemon::stop(self),
+            Some("lsp_proxy") => commands::daemon::lsp_proxy(),
```

Comment:
I tried a couple of different CLI tools and it seems to be uncommon to have multi-word commands. But I found an example in Rome classic and in `gh`: both are using dashes to join the worlds: 

```suggestion
            Some("lsp-proxy") => commands::daemon::lsp_proxy(),
```